companion/src/firmwares/boards.cpp

```diff
@@ -869,7 +873,7 @@ int Boards::getDefaultInternalModules(Board::Type board)
   case BOARD_JUMPER_TLITE:
   case BOARD_RADIOMASTER_ZORRO:
   case BOARD_JUMPER_TPRO:
-    return (int)MODULE_TYPE_MULTIMODULE;
+    return (int)MODULE_TYPE_CROSSFIRE;
```

Comment:
I don’t think this is the case for T-lite.

Reply:
Hi, I changed that for TPro with internal ELRS, which was recently released and I thought it will be the most attractive version in the future.

I can surely change it back to Multimodule if it's a better option.

Reply:
You're right, going to correct this.

Reply:
You can probably break the Zorro out on it's own since it's the only one we build as crossfire by default atm... and leave the others as multimodule.

Reply:
You can probably break the Zorro out on it's own since it's the only one we build as crossfire by default atm... and leave the others as multimodule.

Reply:
OK, did it as you suggested.